torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1462,6 +1477,26 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                SkipInfo('TestCommon', 'test_variant_consistency_jit',
                         dtypes=[torch.bfloat16, torch.float16, torch.cfloat, torch.cdouble]),),
            sample_inputs_func=sample_inputs_addmm),
+    OpInfo('amax',
+           op=torch.amax,
+           dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           dtypesIfCPU=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16, torch.bool),
+           test_inplace_grad=False,
+           sample_inputs_func=sample_inputs_amax_amin,
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_jit',
```

Comment:
Could you add a comment with the error you get from this test?

Reply:
Oops, I forgot to add it. Thanks for catching this!

Reply:
Hello @heitorschueroff, I had added the comments. Thank you!